addons/finger/code/fn__comm.sqf

```diff
@@ -0,0 +1,94 @@
+#include "addon.hpp"
+#include "main.hpp"
+
+// finger-pointing array format:
+//   1: source player
+//   2: position of finger-pointing
+//   3: players to broadcast the finger-pointing to
+
+// server-side event-handlers
+	
+if (isServer) then
+{
+	// receives finger-pointing array and broadcasts it to provided players
+	
+	GVAR(fnc_pv_fpToServer) = 
+	{
+		LOGV(_this);
+
+		PV(_val) = _this select 1;
+		
+		PV(_sp) = _val select 0;
```

Comment:
If possible avoid non-standard macros like this one, which makes the code less readable